### PR TITLE
Add instabook to portfolio

### PR DESCRIPTION
**Pull request by josepmartins**
screenshot 2019-01-19 at 21 37 07

screenshot 2019-01-19 at 21 59 52

### CODE REVIEW

**Comment by hihayk** [comment]
Forgot to import Open Sans?

File: gallery/instabook/main.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+.material-icons.blueViolet { color: blueviolet; }
+.material-icons.orange { color: orange; }
+.material-icons.turquoise { color: turquoise; }
+.material-icons.blue { color: royalblue ; }
+.material-icons.red { color: tomato ; }
+
+
+@media (max-width: 720px) {
+	.mainSection {
+		max-width: var(--space-4sxl);
```
Comment (hihayk) [inline gallery/instabook/main.css]: Should this be `--space-4xl`?

**Comment by hihayk** [comment]
Can you remove `.DS_Store`files?

File: gallery/instabook/main.css (hunk 1 above)
Comment (josepmartins) [reply gallery/instabook/main.css]: done!

**Review by hihayk — APPROVED** [review]
❤️

**Pull request merged**